### ISSUE TITLE
Custom marked configuration

### ISSUE THREAD

**Issue by Mylloon** [xy2z/PineDocs#229]
Hello, it would be handy if I could change the marked options in the `config.yaml` file or like the one in `mathjax.js`.

**Comment by xy2z** [comment]
Agreed. It would be more customizable if its in its own config file like the mathjax
